README.md

```diff
@@ -77,6 +77,17 @@ pip install tmhmm.py
 ~~~~
 git clone https://github.com/transXpress/transXpress-snakemake.git .
 ~~~~
+or create transXpress folder and checkout the transXpress code into it
```

Comment:
Actually you can just remove the dot: `git clone https://github.com/transXpress/transXpress-snakemake.git` will automatically create the transXpress-snakemake folder and check out the repository into it.